src/frontends/tensorflow_lite/tests/test_models/gen_scripts/generate_pose_detector.py

```diff
@@ -18,7 +19,11 @@
     os.makedirs(path_to_model_dir, exist_ok=True)
 if not os.path.exists(tflite_model_path):
     if not os.path.exists(task_file_path):
-        urllib.request.urlretrieve("https://storage.googleapis.com/mediapipe-models/pose_landmarker/pose_landmarker_lite/float16/latest/pose_landmarker_lite.task", task_file_path)
+        try:
+            urllib.request.urlretrieve(src_url, task_file_path)
```

Comment:
TypeError: a bytes-like object is required, not 'str'

Reply:
looks like it is msg by another generator, if this pr fails - I'll fix it here, if not - as a separate pr

Reply:
https://github.com/openvinotoolkit/openvino/pull/21686